src/StoreApi/docs/cart.md

```diff
@@ -471,8 +471,6 @@ Returns the full [Cart Response](#cart-response) on success, or an [Error Respon
 
 <!-- FEEDBACK -->
 
```

Comment:
These files still have some unrelated changes for some reason.

Reply:
Sorry about this, seems like some rogue *.md changes spilled into this PR, not sure what happened. Reverted all of them 👍🏼